ivy_tests/test_ivy/test_functional/test_core/test_creation.py

```diff
@@ -1,157 +1,404 @@
 """Collection of tests for creation functions."""
 
 # global
+from enum import Flag
 import numpy as np
 import pytest
 from numbers import Number
 from hypothesis import given, strategies as st
 
 # local
 import ivy
+from ivy.functional.backends.numpy.data_type import dtype
 import ivy_tests.test_ivy.helpers as helpers
 import ivy.functional.backends.numpy as ivy_np
 
 
-# array
+# native_array - DONE
```

Comment:
best to remove comments to yourself from the commit, such as `DONE` in this case.